storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -409,6 +409,13 @@ static void get_env_args() {
   get_env_value(opt_swift_storage_url, "OS_STORAGE_URL");
   get_env_value(opt_cacert, "OS_CACERT");
 
+  get_env_value(opt_s3_access_key, "S3_ACCESS_KEY_ID");
```

Comment:
add a comment that these should be always above the "AWS_*" and shouldn't be moved because the order of perference 1->2->3.. (subsitute numbers with actual access types)